velodyne_pointcloud/src/lib/rawdata.cc

```diff
@@ -269,6 +309,12 @@ inline float SQR(float val) { return val*val; }
     using velodyne_pointcloud::LaserCorrection;
     ROS_DEBUG_STREAM("Received packet, time: " << pkt.stamp);
 
+    /** special parsing for the VLS128 **/
+    if (pkt.data[1205] == 161 || pkt.data[1205] == 99) { // VLS 128
```

Comment:
What do these values mean? Is there somewhere they are documented?

Reply:
According to the manual the Byte in the position 1205 of the packet  corresponds to the Product ID. In the case of the VLS-128 the ID is 161. I can not find the 99 in the manual so I will remove it from the condition and make the 161 a constant in rawdata.h

Reply:
That's fine. Can you please add a comment in the code explaining this when you make the change?